solutions/remotemonitoring/single-vm/docker-compose.java.yml

```diff
@@ -198,7 +198,7 @@ services:
       - PCS_TELEMETRY_STORAGE_TYPE
 
   diagnostics:
-    image: azureiotpcs/pcs-diagnostics-dotnet:${PCS_DOCKER_TAG}
+    image: azureiotpcs/pcs-diagnostics-dotnet:Diagnostics-2.0.0
```

Comment:
Is this going to be new Diagnostics service version?

Reply:
yes